readthedocs/forms.py

```diff
@@ -0,0 +1,63 @@
+"""Community site-wide form overrides."""
+
+import json
+import logging
+
+import requests
+
+from allauth.account.forms import SignupForm
+from django.conf import settings
+from django.forms.fields import BooleanField
+
+log = logging.getLogger(__name__)  # noqa
+
+
+class SignupFormWithNewsletter(SignupForm):
+
+    """Custom signup form that includes a checkbox to subscribe to a newsletter."""
+
+    receive_newsletter = BooleanField(
+        required=False,
+        label=(
+            "I also wish to subscribe to the Read the Docs newsletter "
+            "to receive news and updates regularly. "
+            "I know I can unsubscribe at any time."
+        ),
+    )
+
+    field_order = [
+        'email',
+        'username',
+        'password1',
+        'password2',
+        'receive_newsletter',
+    ]
+
+    def save(self, request):
+        user = super().save(request)
+
+        if self.cleaned_data.get("receive_newsletter"):
+            log.debug('Subscribing user to newsletter')
```

Comment:
```suggestion
            log.info('Subscribing user to newsletter. email=%s', self.cleaned_data["email"])
```

It may be good to log this as INFO. I'm not sure if the users' email or username can be logged as well, but we can remove it otherwise.